README.md

```diff
@@ -13,7 +13,7 @@ The plugin then instantiates a Wasm component and injects the message to the com
 Install MQTT Plugin:
 
 ```
-spin plugin install --url https://raw.githubusercontent.com/spinkube/spin-trigger-mqtt/main/trigger-mqtt-remote.json --yes
+spin plugin install --url https://github.com/spinkube/spin-trigger-mqtt/releases/download/v0.2.0/trigger-mqtt.json --yes
```

Comment:
I do not believe we have v0.1.0 released via CI, so we could possibly just start with that and then start adding it to the plugins repository so that it is more discoverable?

Reply:
I was just using the version listed in pluginify.toml but happy to change this to whatever the first tag should be. I can change it to canary for the time being since that should be available after the PR is merged?